core/src/components/button/button.ionic.scss

```diff
@@ -172,6 +174,25 @@
   aspect-ratio: 1 / 1;
 }
 
+// Button with Spinner
+// -------------------------------------------------------------------------------
+::slotted(ion-spinner) {
+  --color: currentColor;
+}
+
+/* Button containing only a spinner */
+::slotted(ion-spinner[slot="start"]),
+::slotted(ion-spinner[slot="end"]),
+::slotted(ion-spinner[slot="icon-only"]) {
+  width: globals.$ion-space-500;
+  height: globals.$ion-space-500;
+}
+
+:host(.button-xlarge) ::slotted(ion-spinner) {
+  width: globals.$ion-space-600;
+  height: globals.$ion-space-600;
+}
```

Comment:
These don't seem to be consistent with design:
- Small has `space-600` if there's text
- Small has `space-500` if there's no text
- Medium has `space-500` regardless of text
- Large has `space-500` regardless of text

Please verify if designs are out of date or if it's the code.